monai/metrics/regression.py

```diff
@@ -10,23 +10,38 @@
 # limitations under the License.
 
 import math
-from abc import ABC, abstractmethod
+from abc import abstractmethod
 from functools import partial
-from typing import Any, Union
+from typing import Any, Optional, Union
 
 import torch
 
 from monai.metrics.utils import do_metric_reduction
 from monai.utils import MetricReduction
 
+from .metric import Metric
+
+
+class RegressionMetric(Metric):
+    """
+    Base class for regression metrics.
+    Input `y_pred` (BCHW[D] where C is number of channels) is compared with ground truth `y` (BCHW[D]).
```

Comment:
same comments in `monai/metrics/hausdorff_distance.py`

Reply:
updated in the latest commit.

Thanks.